src/s2wasm.h

```diff
@@ -1167,6 +1173,42 @@ class S2WasmBuilder {
         }
       }
     }
+    if (startFunction.size()) {
```

Comment:
Sorry, I wasn't asking about that line. I mean this entire part of the diff.

IOW I have no idea what this pull request actually does, since this is the main part! :)